examples/multimodal/webvid/scripts/bench_inside.py

```diff
@@ -8,7 +8,7 @@
 
 import numpy as np
 
-from streaming.multimodal.webvid import StreamingInsideWebVid
+from examples.multimodal.webvid.read import StreamingInsideWebVid
```

Comment:
Can we make sure that `examples` is included when installing the `mosaicml-streaming` package, like in a new venv or something? Ran into a similar issue with the simulator and want to make sure that the reorg doesn't affect this as well.

Reply:
Oh yeah, was worrying about that. How did you handle this wrt simulator?

Tentatively, since I am repurposing `examples/`, you would think it should be safe, and `notebooks/` and `benchmarks/` might be a problem.

Following up, when I look at `setup.py`, there is this line
```
    packages=setuptools.find_packages(exclude=['tests*']),
```
which seems to suggest we're fine? What say you.